Gemfile.lock

```diff
@@ -82,4 +82,4 @@ DEPENDENCIES
   rdoc (>= 4.3)
 
 BUNDLED WITH
-   1.17.2
+   2.1.2
```

Comment:
This was needed because of failures with Ruby 2.7.
https://travis-ci.org/plataformatec/mail_form/jobs/631885865?utm_medium=notification&utm_source=github_status

Tests are green after this change:
https://travis-ci.org/plataformatec/mail_form/jobs/631893557?utm_medium=notification&utm_source=github_status